src/app/[locale]/(pages)/blocco-accesso/magic-link/page.tsx

```diff
@@ -21,6 +25,7 @@ const ExpiredMagicLink = () => {
       pushWithLocale(ROUTES.INTERNAL_ERROR);
     } else {
       storageTokenOps.write(token);
+      storageMagicLinkOps.write({ value: true });
```

Comment:
How is this related with this PR?

Reply:
Here, we introduced this because we needed a discriminator to determine whether we are entering with a mail link or not, as we have different Mixpanel events on the profile page